### PR TITLE
In case, we do not build Fortran, Fortran 2008 or CXX, the regexp in …

### PR DESCRIPTION
**Pull request by hpcraink**
…make_manpage.pl will delete all lines up to the next ".fi" -- which for functions that do not implement the corresponding interface "as code" will have all text eliminated. Change regexp to delete this language-specific content up to the next newline+section header ".SH"

Also in case of make V=1, we'd like to see the command line, too (see Makefile.ompi-rules)

Amend OMPI_Affinity_str according to the other man-pages definitions.

### CODE REVIEW

**Comment by hpcraink** [comment]
@ggouaillardet Could You please have a look at the changes in `make_manpage.pl`? Thanks.

**Comment by ggouaillardet** [comment]
@hpcraink i am fine with this PR, thanks for filling it.

out of curiosity, did you run into the trouble you fixed (e.g. all text deleted) ?
at first glance, the root cause seems there is a missing ```.fi``` in the man page ?

**Comment by hpcraink** [comment]
@ggouaillardet Thanks!
Well, some man-pages, that did not define a Fortran (Or C++ Interface) were lacking any content when shown... Took me a while to figure out, that it was the parsing through the Script, and that it's the code section missing; Now I think the \n.SH assumption is fine.

File: Makefile.ompi-rules [hunk 1]
```diff
@@ -21,7 +21,7 @@ endif
 
 .1in.1:
 	@ echo "  BUILDING $@"
-	@ $(top_srcdir)/ompi/mpi/man/make_manpage.pl \
+	$(AM_V_CC) $(top_srcdir)/ompi/mpi/man/make_manpage.pl \
```
Comment (jsquyres) [inline Makefile.ompi-rules]: I think this should be `$(OMPI_V_GEN)`.  Shouldn't the `echo` statement be removed, too?

**Comment by ibm-ompi** [comment]
Test passed.

**Comment by jsquyres** [comment]
bot:retest

**Pull request merged**